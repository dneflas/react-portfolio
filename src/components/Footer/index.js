import React from "react";

function Footer() {
  return (
    <div className="flex-row justify-space-between my-3">
      <div className="col-md-4 col-lg-2 text-center">
        <a
          href="mailto:debbieneflas@gmail.com"
          className="ml-2 my-1 px-2 py-1 text-dark"
        >
          <i className="fa-solid fa-envelope"></i>
        </a>
      </div>
      <div className="col-md-4 col-lg-2 text-center">
        <a href="tel:+16263771363" className="ml-2 my-1 px-2 py-1 text-dark">
          <i className="fa-solid fa-phone"></i>
        </a>
      </div>
      <div className="col-md-4 col-lg-2 text-center">
        <a
          href="https://github.com/dneflas"
          className="ml-2 my-1 px-2 py-1 text-dark"
          target="_blank"
          rel="noopener noreferrer"
        >
          <i className="fa-brands fa-github"></i>
        </a>
      </div>
      <div className="col-md-4 col-lg-2 text-center">
        <a
          href="https://www.linkedin.com/in/debbieneflas/"
          className="ml-2 my-1 px-2 py-1 text-dark"
          target="_blank"
          rel="noopener noreferrer"
        >
          <i className="fa-brands fa-linkedin"></i>
        </a>
      </div>
      <div className="col-md-4 col-lg-2 text-center">
        <a
          href="https://stackoverflow.com/"
          className="ml-2 my-1 px-2 py-1 text-dark"
          target="_blank"
          rel="noopener noreferrer"
        >
          <i className="fa-brands fa-stack-overflow"></i>
        </a>
      </div>
    </div>
  );
}
export default Footer;
