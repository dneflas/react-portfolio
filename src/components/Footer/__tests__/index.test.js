import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "..";

afterEach(cleanup);

describe("Footer components render", () => {
  it("renders", () => {
    render(<Footer />);
  });
  it("matches snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
