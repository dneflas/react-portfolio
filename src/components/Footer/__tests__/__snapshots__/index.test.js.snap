// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer components render matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex-row justify-space-between my-3"
  >
    <div
      class="col-md-4 col-lg-2 text-center"
    >
      <a
        class="ml-2 my-1 px-2 py-1 text-dark"
        href="mailto:debbieneflas@gmail.com"
      >
        <i
          class="fa-solid fa-envelope"
        />
      </a>
    </div>
    <div
      class="col-md-4 col-lg-2 text-center"
    >
      <a
        class="ml-2 my-1 px-2 py-1 text-dark"
        href="tel:+16263771363"
      >
        <i
          class="fa-solid fa-phone"
        />
      </a>
    </div>
    <div
      class="col-md-4 col-lg-2 text-center"
    >
      <a
        class="ml-2 my-1 px-2 py-1 text-dark"
        href="https://github.com/dneflas"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          class="fa-brands fa-github"
        />
      </a>
    </div>
    <div
      class="col-md-4 col-lg-2 text-center"
    >
      <a
        class="ml-2 my-1 px-2 py-1 text-dark"
        href="https://www.linkedin.com/in/debbieneflas/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          class="fa-brands fa-linkedin"
        />
      </a>
    </div>
    <div
      class="col-md-4 col-lg-2 text-center"
    >
      <a
        class="ml-2 my-1 px-2 py-1 text-dark"
        href="https://stackoverflow.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          class="fa-brands fa-stack-overflow"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
